### PR TITLE
Room 1: plotting script added

### PR DESCRIPTION
**Pull request by leriomaggio**

### CODE REVIEW

**Review by dokato — COMMENTED** [review]
crappy code

File: snippets/plotting.py [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+import matplotlib.pyplot as plt
+import pandas as pd
+
+from typing import Union
+from itertools import filterfalse
+from pathlib import Path
+from textwrap import wrap
+
+# Chart Data Folder
+CHART_FOLDER = Path("/home/ggk765/Analysis/Charts")
```
Comment (dokato) [inline snippets/plotting.py:13]: this is particularly 💩

Comment (leriomaggio) [reply snippets/plotting.py:13]: Totally agree!

File: snippets/plotting.py [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+if not os.path.exists(CHART_FOLDER):
+    os.makedirs(CHART_FOLDER, exist_ok=True)
+
+default_color_map = plt.get_cmap("Set2")
```
Comment (dokato) [inline snippets/plotting.py:17]: What a spectacular function call :O

Comment (leriomaggio) [inline snippets/plotting.py:17]: this should be UPPER case as it's a constant

**Pull request closed without merging**